vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -175,21 +175,26 @@ public void selectEditorOptionsAreSet() {
         Assert.assertTrue(optionsList.contains("Sales"));
     }
 
+    @Test
+    public void disabledGridShouldNotBeActivatedByDoubleClick() {
```

Comment:
The test isn't valid. It passes even without the change in GridPro.java

Reply:
Done.